qa/L0_trace/opentelemetry_unittest.py

```diff
@@ -121,32 +153,38 @@ def _check_parent(self, child_span, parent_span):
         # Check that child and parent span have the same trace_id
         # and child's `parent_span_id` is the same as parent's `span_id`
         self.assertEqual(child_span["trace_id"], parent_span["trace_id"])
-        self.assertIn(
-            "parent_span_id",
-            child_span,
+        self.assertNotEqual(
+            child_span["parent_span_id"],
+            "0000000000000000",
```

Comment:
Can this be a descriptive variable instead of magic number? Not sure what all zeroes or number of zeroes importance is.

Reply:
When parent for span is not specified, `parent_span_id` field equal to "0000000000000000".